TestApp/ios/Pods/Target Support Files/Pods-TestApp/Pods-TestApp-acknowledgements.markdown

```diff
@@ -28,20 +28,4 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
-
```

Comment:
what happened here?

Reply:
This happened again when I ran the command `./update-link-to-appcenter.sh`